sdk/nodejs/package.json

```diff
@@ -27,7 +27,7 @@
         "fetch-retry": "^5.0.6"
     },
     "peerDependencies": {
-        "@openfeature/server-sdk": "^1.11.0"
+        "@openfeature/server-sdk": "*"
```

Comment:
are we okay to explicitly  make it `*` and if so should the root `project.json` use the same or we want different strategies on purpose?

Reply:
yea I made that change a while ago on this PR, not certain why. Will likely revert it.